packages/contracts-bedrock/scripts/upgrades/holocene/README.md

```diff
@@ -0,0 +1,28 @@
+# Holocene Upgrade
+
+This directory contains a repeatable task for upgrading an `op-contracts/v1.6.0` deployment to `op-contracts/v1.8.0`.
```

Comment:
Per https://discord.com/channels/1244729134312198194/1266465563568509058/1305784053487894538, can we expand this README with docs that explain how to use this script that differ based on whether you're on `op-contracts/v1.6.0` vs. `op-contracts/v1.3.0`?

Reply:
Anyone running `v1.3.0` should upgrade to `v1.6.0` before doing this upgrade.

Reply:
Why is that? Per https://discord.com/channels/1244729134312198194/1304003492783849512/1305783304548651058 the plan is to release a version of 1.8.0 with L2OO support, which is also known as 1.3.0 with the 1.8.0 SystemConfig

Reply:
@mds1 is correct, the intention here is to also allow for purely deploying and upgrading the `SystemConfig`. This is covered in the updates I've done this morning, separating the dispute contracts' upgrade from the `SystemConfig` upgrade.

When we actually execute the upgrade, we'll have to do it in two parts anyways for safety. If we upgraded the `SystemConfig` prior to the hardfork, an accidental or uninformed call to `setEip1559Params` could cause the chain to halt due to an unexpected `UpdateType` in the `ConfigUpdate` event.

This means that our order of operations is:
1. Upgrade dispute contracts (optional).
2. Activate Holocene
3. Upgrade `SystemConfig` contract.

For those who are upgrading from `op-contracts/v1.3.0` -> `op-contracts/v1.8.0-deprecated-l2oo` (or whatever we're calling that configuration), they would _only_ upgrade the `SystemConfig` after fork activation.